src/ray/common/task/task_util.h

```diff
@@ -86,7 +86,7 @@ class TaskSpecBuilder {
       const rpc::Address &caller_address, uint64_t num_returns,
       const std::unordered_map<std::string, double> &required_resources,
       const std::unordered_map<std::string, double> &required_placement_resources,
-      const PlacementGroupID &placement_group_id,
+      const PlacementGroupID &placement_group_id, const int64_t bundle_index,
```

Comment:
1) use a pair for placement_group_id and bundle_index, because they should always be set together.
2) give it a default value to avoid changing all call sites.